pkg/security/secl/go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/stretchr/testify v1.9.0
 	golang.org/x/sys v0.18.0
 	golang.org/x/text v0.14.0
-	golang.org/x/tools v0.17.0
+	golang.org/x/tools v0.19.0
```

Comment:
Because of https://github.com/DataDog/datadog-agent/pull/24080, we cannot bump this